pkg/resmap/resmap.go

```diff
@@ -206,6 +207,9 @@ func newResourceSliceFromBytes(in []byte) ([]*resource.Resource, error) {
 		var out unstructured.Unstructured
 		err = decoder.Decode(&out)
 		if err != nil {
+			if strings.Contains(err.Error(), "is missing in 'null'") {
```

Comment:
This works but i find it hard to read.
Perhaps document the error with a method name, e.g.

```
var err error
for err == nil || isEmptyYamlError(err) {
  var out unstructured.Unstructured
  err = decoder.Decode(&out)
  if err == nil {
    result = append(result, resource.NewResourceFromUnstruct(out))
  }
}
```
where
```
func isEmptyYamlError(err) bool { return strings.Contains(err.Error(), "is missing in 'null'") }
```

Reply:
updated.